XP-Pen_Artist_156/descriptors.txt

```diff
@@ -0,0 +1,130 @@
+
```

Comment:
I think the "identifier" for this tablet should have the dot preserved i.e. be `XP-Pen_Artist_15.6`. So both the directory and the photo should have it too.